Gordon360/Controllers/NewsController.cs

```diff
@@ -22,6 +22,7 @@ namespace Gordon360.Controllers
     public class NewsController : GordonControllerBase
     {
         private readonly INewsService _newsService;
+        private IEnumerable<AuthGroup> user_groups { get; set; }
```

Comment:
I don't think you want `user_groups` to be a Property of the NewsController. That would imply setting it when `NewsController` is constructed, but you don't know anything about the requesting user at that point.

Instead, this should be a variable scoped to each request that needs to know that auth groups of the requesting user.